soc/nxp/imx/imx9/imx943/Kconfig.defconfig.mimx94398.a55

```diff
@@ -0,0 +1,28 @@
+# Copyright 2025 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_MIMX94398_A55
+
+# Workaround for not being able to have commas in macro arguments
+DT_CHOSEN_Z_FLASH := zephyr,flash
+
+config FLASH_SIZE
+	default $(dt_chosen_reg_size_int,$(DT_CHOSEN_Z_FLASH),0,K)
+
+config FLASH_BASE_ADDRESS
+	default $(dt_chosen_reg_addr_hex,$(DT_CHOSEN_Z_FLASH))
+
+# Enable GIC Safe Configuration to run multiple OSes on Cortex-A Cores
+config GIC_SAFE_CONFIG
+	default y
+
+config MCUX_CORE_SUFFIX
+	default "_ca55" if SOC_MIMX94398_A55
+
+config NUM_IRQS
+	default 320
+
+config SYS_CLOCK_HW_CYCLES_PER_SEC
+	default 24000000
```

Comment:
polite reminder that this needs to come from dts in all future PRs, will allow here since PR was opened earlier

Reply:
thanks, will keep it in this PR.